mail/spring-cloud-starter-stream-source-mail/src/main/resources/META-INF/spring.provides

```diff
@@ -0,0 +1 @@
+provides: spring-cloud-starter-stream-source-mail
```

Comment:
New line in the end of each file.
The IDE has a configuration option on the matter.
Yes, it is minor, too, but that saves some time for all of us.
At least I won't write such a comment and you won't read :smile: 

Just some code style rule.